### PR TITLE
Extrude From Multiple Extruders On an LCD Controller

### PR DESCRIPTION
**Pull request by gralco**
Title says it all.

### CODE REVIEW

**Comment by oysteinkrog** [comment]
Please fix the commit message to describe the changes.

File: Marlin/ultralcd.cpp [hunk 1]
```diff
@@ -752,8 +760,31 @@ static void lcd_move_e() {
...
+#if EXTRUDERS > 1
+  static void lcd_move_e1() { lcd_move_e(1) }
+  #if EXTRUDERS > 2
+    static void lcd_move_e2() { lcd_move_e(2) }
+    #if EXTRUDERS > 3
+      static void lcd_move_e3() { lcd_move_e(3) }
+    #endif
+  #endif
+#endif // EXTRUDERS > 1
+
```
Comment (thinkyhead) [inline Marlin/ultralcd.cpp]: Am I seeing double??

**Comment by thinkyhead** [comment]
Looks great, except the one doubled-up part. Will merge as soon as that's fixed.

**Comment by gralco** [comment]
@thinkyhead
Thanks for reviewing and nice catch; it's been fixed.

**Pull request merged**